pkg/kubelet/apis/podresources/server_v1_test.go

```diff
@@ -710,12 +710,8 @@ func TestGetPodResourcesV1(t *testing.T) {
 					t.Errorf("want exit = %v, got %v", tc.err, err)
 				}
 			} else {
-				if err != err {
```

Comment:
I guess the original intention was to compare `err` with `tc.err` here. WDYT?

Reply:
It seems that the logic in line 708 with `if err != nil` has already compared err with tc.err, and this logic is inside else, so err is nil, so there is no need to compare err. Just comparing `tc.expectedResponse `with `resp` is enough.

Reply:
There is a need to catch false positives, i.e. situations with expected non-nil error (tc.err) and returned nil error.
Does this make sense to you?